docs/sources/userguide/plugins.md

```diff
@@ -0,0 +1,157 @@
+Docker Plugins MVP
+==================
+
+Plugins provide a mechanism for hooking into the Docker engine to extend its behavior.
+
+HTTP over a local UNIX socket is used to communicate between Docker and its plugins.
+
+Plugins are distributed as containers.
```

Comment:
Also against permissions. But I can't stress enough how important the Docker socket is here. I don't think it should be included by default, but I do really really really want a convenience flag for it. I don't care if it is someday deprecated or removed, assuming it is replaced by something else that works (limited permissions, etc)